packages/components/package.json

```diff
@@ -76,7 +76,7 @@
     "vis-network": "~6.5.2"
   },
   "devDependencies": {
-    "@labkey/build": "~6.1.6",
+    "@labkey/build": "~6.2.0-fb-es-modules.1",
```

Comment:
note: the `~` didn't work here for me with the alpha package version. I had to remove that to see the actual changes pulled down for this version.